sql-plugin/src/main/spark340/scala/com/nvidia/spark/rapids/shuffle/RapidsShuffleIterator.scala

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2020-2023, NVIDIA CORPORATION.
+ * Copyright (c) 2023, NVIDIA CORPORATION.
```

Comment:
it looks like this removed copyright when it was move,  is that just a diff issue?

Reply:
This was shimmed actually. This file should only have 2023. It was accidentally removed in the other file (spark311 version).